### PR TITLE
Handled exception of`--file` option with non-existed file(#4047)

### PR DESCRIPTION
**Pull request by leehwarang**
### Description of the Change

In **handleRequires**, a function that handles exceptions when invoking a module that does not exist with the `--require` option. However, function **handleRequires** can also be used to load non-existent files with the `--file` option.

So I added this function to be called when I loaded the file using --file option. And I modified the annotation of this function.

>./bin/mocha --require non-exist-module

module

>./bin/mocha --file non-exist-file

file

### Benefits

When user import a non-existent file or module, command line can show the user the same error message and format.

### CODE REVIEW

**Comment by outsideris** [comment]
ref: #4047

File: lib/cli/run-helpers.js [hunk 1]
```diff
@@ -91,6 +92,19 @@ exports.handleRequires = (requires = []) => {
...
 
+/**
+ * @param {string[]} files - Modules to require by `--file <file>`
+ * @private
+ */
+
+exports.handleFiles = (files = []) => {
+  files.forEach(file => {
+    if (!fs.existsSync(file, {cwd})) {
+      throw Error(`Cannot find module '${file}'`);
```
Comment (craigtaub) [inline lib/cli/run-helpers.js]: Could you add a test for this please.

File: lib/cli/run-helpers.js [hunk 2]
```diff
@@ -79,6 +79,7 @@ exports.list = str =>
  * @param {string[]} requires - Modules to require
  * @private
  */
+
```
Comment (craigtaub) [inline lib/cli/run-helpers.js]: Doesnt need the space (same below)

**Comment by JoshuaKGoldberg** [comment]
🤖 Closing out old PRs to keep the queue small. Please yell at me if you'd like to keep working on this. Cheers!

**Pull request closed without merging**